cadctl/cmd/investigate/investigate.go

```diff
@@ -239,7 +245,19 @@ func clusterRequiresInvestigation(cluster *cmv1.Cluster, ocmClient *ocm.SdkClien
 	}
 
 	if !isAWSCluster {
-		return false, utils.EscalateAlertIfNotLS("CAD could not run an automated investigation on this cluster: unsupported cloud provider.", cluster, pdClient, ocmClient)
+		hasSupportException := false
+		if supportExValue, err := strconv.ParseBool(cd.Labels[ClusterDeploymentSupportExceptionLabel]); err == nil {
+			hasSupportException = supportExValue
+		}
+
+		ls, err := ocmClient.IsInLimitedSupport(cluster.ID())
+		if err != nil {
+			return false, err
+		}
+
+		if ls && !hasSupportException {
```

Comment:
This change breaks the logic of escalation in case of a GCP alert, with this PR, CAD would try to investigate GCP clusters and run into errors instead of escalating them.